src/main/menu_main.cpp

```diff
@@ -1560,6 +1562,62 @@ static void s_drawGameSaves()
         A++;
         SuperPrint("ERASE SAVE", 3, 300, 320 + (A * 30));
     }
+
+    if(MenuCursor < 0 || MenuCursor >= maxSaveSlots || (MenuMode != MENU_SELECT_SLOT_1P && MenuMode != MENU_SELECT_SLOT_2P) || SaveSlotInfo[MenuCursor + 1].Progress < 0)
+        return;
+
+    const auto& info = SaveSlotInfo[MenuCursor + 1];
+
+    int infobox_x = ScreenW / 2 - 240;
+    int MenuY = ScreenH - 250;
+    int infobox_y = MenuY + 145;
+
+    int row_1 = infobox_y + 10;
+    int row_2 = infobox_y + 42;
+    int row_c = infobox_y + 26;
+
+    // recenter if single row
+    if((info.Time <= 0 || g_speedRunnerMode == SPEEDRUN_MODE_OFF) && !gEnableDemoCounter)
+        row_1 = infobox_y + 26;
+
+    XRender::renderRect(infobox_x, infobox_y, 480, 68, 0, 0, 0, 0.5f);
+
+    // Temp string
+    std::string t;
+
+    // Score
+    bool show_timer = info.Time > 0 && g_speedRunnerMode != SPEEDRUN_MODE_OFF;
+    int row_score = show_timer ? row_1 : row_c;
+    SuperPrint(t = fmt::format_ne("Score: {0}", info.Score), 3, infobox_x + 10, row_score);
+
+    // Gameplay Timer
+    if(show_timer)
+    {
+        std::string t = GameplayTimer::formatTime(info.Time);
+
+        if(t.size() > 9)
+            t = t.substr(0, t.size() - 4);
+
+        SuperPrint(fmt::format_ne("Time: {0}", t), 3, infobox_x + 10, row_2);
+    }
+
+    // If demos off, put (l)ives and (c)oins on center
+    int row_lc = (gEnableDemoCounter) ? row_1 : row_c;
+
+    // Print lives on the screen (from gfx_update2.cpp)
+    XRender::renderTexture(infobox_x + 272, row_lc + 2 + 14 - GFX.Interface[3].h, GFX.Interface[3]);
+    XRender::renderTexture(infobox_x + 272 + 40, row_lc + 2 + 16 - GFX.Interface[3].h, GFX.Interface[1]);
+    SuperPrint(t = std::to_string(info.Lives), 1, infobox_x + 272 + 62, row_lc + 2);
+
+    // Print coins on the screen (from gfx_update2.cpp)
+    int coins_x = infobox_x + 480 - 10 - 36 - 62;
+    XRender::renderTexture(coins_x + 16, row_lc, GFX.Interface[2]);
+    XRender::renderTexture(coins_x + 40, row_lc + 2, GFX.Interface[1]);
+    SuperPrintRightAlign(t = std::to_string(info.Coins), 1, coins_x + 62 + 36, row_lc + 2);
+
+    // Fails Counter
+    if(gEnableDemoCounter)
+        SuperPrintRightAlign(fmt::format_ne("{0}: {1}", gDemoCounterTitle, info.Fails), 3, infobox_x + 480 - 10, row_2);
```

Comment:
Little note that the demos counter can be enabled at the episode side too via `compat.ini` :eyes: